src/pcre2_compile_class.c

```diff
@@ -245,6 +245,7 @@ while (c <= end)
     buffer[0] = new_start;
     buffer[1] = new_end;
     buffer += 2;
+    (void)buffer;
```

Comment:
We have some clearly useless assignments here, but they're OK to keep, because they ensure the buffer is correctly tracked if someone extends the function later. Easy suppression.